src/scss/dp/overrides/design-system/_components.scss

```diff
@@ -8,6 +8,9 @@
     padding-bottom: 0;
   }
   &-table {
+    &__header.ons-u-pb-s {
```

Comment:
I really don' like this kind of class names `ons-u-pb-s`. No idea what it means.  can you maybe use something meaningfull, like `with bottom-padding` which will not contain specific names of component so they are more generic and can be re used

Reply:
These are [design-system](https://github.com/ONSdigital/design-system) utility classes and the reason I am putting them in this way is because their utility classes sometimes do and do not include `!important` so I have to come up with a way of increasing the specificity of my css to deliver the design but not make it too confusing to someone who comes along later on to maintain it 

And `ons-u-pb-s` means: domain-utility-paddingBottom-small

Reply:
ok, thank you. got it :-)